app/models/follow.rb

```diff
@@ -0,0 +1,9 @@
+class Follow
+  
+  attr_reader :login, :html_url
+
+  def initialize(attributes)
+    @login = attributes[:login]
+    @html_url = attributes[:html_url]
+  end
+end
```

Comment:
Good call making one Follow object that we can leverage for both followers and following.